com.unity.render-pipelines.core/Runtime/Debugging/DebugUpdater.cs

```diff
@@ -31,14 +31,41 @@ static void RuntimeInit()
                 // component is initialized while the GameObject is active. So we deactivate it temporarily.
                 // See https://fogbugz.unity3d.com/f/cases/1323566/
                 go.SetActive(false);
-                go.AddComponent<InputSystemUIInputModule>();
+                var uiModule = go.AddComponent<InputSystemUIInputModule>();
+
+                // FIXME: In order to activate default input actions in player builds (required for touch input to work),
+                // we need to call InputSystemUIInputModule.AssignDefaultActions() which was added in com.unity.inputsystem@1.1.0-pre.5.
+                // However, there is a problem in InputSystem package version ordering, where it sorts this version as an
+                // older version than it should be. Hence we cannot write a version define to conditionally compile this function call.
+                // Instead, we use reflection to see if the function is there and can be invoked.
+                //
+                // Once com.unity.inputsystem@1.1.0 is available, create an INPUTSYSTEM_1_1_0_OR_GREATER version define and use it
+                // to conditionally call AssignDefaultActions().
+                System.Reflection.MethodInfo assignDefaultActionsMethod = uiModule.GetType().GetMethod("AssignDefaultActions");
+                if (assignDefaultActionsMethod != null)
+                {
+                    assignDefaultActionsMethod.Invoke(uiModule, null);
+                }
+
                 go.SetActive(true);
-                EnhancedTouchSupport.Enable();
```

Comment:
This was the main source of the bug that @evelinastaniulyte's testing revealed - if EventSystem component was already found in the scene, touch would not get enabled.